Tangem/App/Factories/SwappingDependenciesFactory/SwappingDependenciesFactoring.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SwappingDependenciesFactoring.swift
+//  Tangem
+//
+//  Created by Sergey Balashov on 26.01.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import TangemExchange
+
+protocol SwappingDependenciesFactoring {
+    // TODO: Should be change to protocol
+    func walletModel() -> WalletModel
+    func userWalletModel() -> UserWalletModel
```

Comment:
```suggestion
    var walletModel: WalletModel { get }
    varuserWalletModel: UserWalletModel { get }
```
Ну и т.д.

или же функция: `func createWalletModel()`. Все же в функции должно быть какое-то действие заложено: создать, предоставить, загрузить, а свойство - что-то хранимое